### ISSUE TITLE
Not supportes with Linux!

### ISSUE THREAD

**Issue by HeaTTheatR** [richarddun/zipper#2]
![2016-12-20 23 49 13](https://cloud.githubusercontent.com/assets/16930280/21367712/9bf8b78c-c718-11e6-86de-a4a8e7dada80.png)

After fix:
![2016-12-21 00 00 24](https://cloud.githubusercontent.com/assets/16930280/21367786/eed4772a-c718-11e6-965d-768d0a107f7d.png)

![2016-12-20 23 59 25](https://cloud.githubusercontent.com/assets/16930280/21367795/fb494c60-c718-11e6-808f-70f3d822e5e9.png)

**Comment by richarddun** [comment]
Thank you for trying this out and leaving feedback!

I was being a bit lazy and only testing on a Windows platform.  I've updated file references now using os.path methods and it launches for me on a non windows platform.  However there are graphical remnants remaining because of a higher resolution in my case - still it should launch ok for you at least.

I'll keep testing for now, let me know if it doesn't at least launch for you.

Cheers 👍

**Comment by richarddun** [comment]
Actually if you left click anywhere on the screen, you 'aim' the character.  Release the left mouse button and it will move quickly in the desired direction.  Hold the mouse button for longer to get a greater distance.  From there you can jump, and use the mouse again ( in mid-air ) :)

It is one of the 'special abilities' I was thinking about for this character, and am currently working on optimising it's design (amongst other things)

**Comment by HeaTTheatR** [comment]
A good engine! Can also be used for other games :)

**Comment by HeaTTheatR** [comment]
However, an empty level: no enemies, no one. That is how it should be?

**Comment by richarddun** [comment]
Yeah, that's as far as I have gotten.  I'm occupied with drawing / designing enemy sprites and additional maps / levels.  What I have seen is that with a very large map size, the tmx handling module I use doesn't seem to scale too well.  I was informed earlier that Kivent (which has tiled support) may be a better choice and that is something I'm looking into as well.

**Comment by spinningD20** [comment]
This is off topic, but do you have anywhere you're documenting your progress?  I'd like to keep tabs on it, since we're using a similar foundation of code.

**Comment by richarddun** [comment]
Hi @spinningD20 actually I have no log or documentation about what I've been looking into so far, apart from memory and musing :)

The long term aim is a move to implement Kivent, but before that I'd like to optimise some of the methods I've already created - there are some pretty bad visual / logic issues in what I've created so far.

Now that you've mentioned it though, I think it may help to start documenting (if for no other reason than to arrange / plan stuff in a coherent way 👍

**Comment by richarddun** [comment]
closing after a long period, but zippy is so back now (thanks to assistance from codex et al).
